wapitiCore/controller/wapiti.py

```diff
@@ -187,6 +186,7 @@ def __init__(self, scope_request: Request, scope="folder", session_dir=None, con
         self.detailed_report = False
         self._headless_mode = "no"
         self._wait_time = 2.
+        self._timeout = 6.
```

Comment:
`_timeout` should not be declared in the `Wapiti` class as a timeout is already stored in the `CrawlerConfiguration` object.